project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.18
+sbt.version=1.5.6
```

Comment:
Should this be 1.5.7?

Reply:
Ah yes, I think when I started on this 1.5.7 wasn't out. Will bump. Thanks.